foundation-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -110,6 +110,410 @@ if (app != null) {
 {% endtab %}
 {% endtabs %}
 
+### Initializing the SDK
```

Comment:
Can we update the Mobile Core API Reference page to list all APIs in alphabetical order by API name like the other extensions do? That makes it easier to navigate through APIs from the right nav panel too. See Adobe Analytics as an example https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference

This way on this page we don't need to load this entire example, but going straight to the point of what registerExtension/registerExtensions APIs do.

Reply:
Will do. What do you think is the best approach for the `registerExtensions` vs `start` APIs since start isn't part of AEP?

Reply:
Yes, I think it makes sense to separate them like that. Example:
registerExtension(s) 
- AEP (3.x) talk about registerExtensions
- ACP (2.x) talk about calling registerExtension for each extension

start
- AEP (3.x) this API also starts the events processing once all extensions are registered, see also registerExtensions (link)
- ACP (2.x) start the events processing, call this API after all extensions have been registered

Reply:
I've restructured this a bit for now. But there still needs to be some clean up in a future task. I will be creating a task for these few left over items.